chessboard.js

```diff
@@ -41,6 +50,32 @@ class Chessboard {
     }
   }
 
+  blackCircle () {
+    return !this.selected.getCircle().isWhite()
+  }
+
+  whiteCircle () {
+    return this.selected.getCircle().isWhite()
+  }
+
+  emptyField (field) {
+    return field.getCircle() === null
+  }
+
+  oneSq (field1, field2) {
+    return [
+      Math.abs(field1.getX() - field2.getX()),
+      Math.abs(field1.getY() - field2.getY())
+    ]
+  }
+
+  moveUp (field1, field2) {
```

Comment:
make this method return true/false

Reply:
return field1.getY() < field2.getY() && field1.getY() - field2.getY() === -1 && Math.abs(field1.getX() - field2.getX()) === 1;